src/test/e2e/21_connect_test.go

```diff
@@ -58,3 +60,40 @@ func TestConnect(t *testing.T) {
 	stdOut, stdErr, err = e2e.Zarf("package", "remove", "init", "--components=logging", "--confirm")
 	require.NoError(t, err, stdOut, stdErr)
 }
+
+func TestMetrics(t *testing.T) {
+	t.Log("E2E: Emits metrics")
+	e2e.SetupWithCluster(t)
+
+	tunnel, err := cluster.NewTunnel("zarf", "svc", "agent-hook", 8888, 8443)
+
+	require.NoError(t, err)
+	err = tunnel.Connect("", false)
+	require.NoError(t, err)
+	defer tunnel.Close()
+
+	// Get untrusted https endpoint
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	client := &http.Client{Transport: tr}
+	https_endpoint := strings.ReplaceAll(tunnel.HTTPEndpoint(), "http", "https")
```

Comment:
lint

Reply:
Also if the mux is serving over https where is it getting the CA cert bundle from?

Reply:
When you say lint what exactly do you mean in this context? Like add a comment explaining or add space between the lines?

Mux is serving the webhook w/ the `tlsCert` and `tlsKey`, which puts the metrics endpoint served at the encrypted https endpoint `/metrics`.  In this case, since it is a client calling the https endpoint through a port-forward we are not getting the CA cert bundle which makes us have to add the [insecure Transport struct](https://github.com/defenseunicorns/zarf/blob/fb18180e215d2f95751001c4587e5139fbe1fdd4/src/test/e2e/21_connect_test.go#L76) to skip certificate verification like doing `curl -k`. Would you instead rather pull the certs to make the request to the metrics endpoint?

Reply:
I think that is actually fine for the test